_includes/nav.html

```diff
@@ -0,0 +1,32 @@
+        <nav class="navbar navbar-default navbar-pf" role="navigation">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse-1">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand" href="{{ site.github.url }}">
+                <img src="img/fabric8.png" alt="Fabric8 Planner User Experience" width="50" style="display: inline-block; vertical-align: bottom;">User Experience Design
+                </a>
+            </div>
+            <div class="collapse navbar-collapse navbar-collapse-1">
+                <ul class="nav navbar-nav navbar-primary">
+                    <li>
+                        <a href="https://github.com/fabric8io/fabric8-ux">Code</a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/fabric8io/fabric8-ux/issues">Issues</a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/fabric8io/fabric8-ux/milestone/5">Backlog</a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/fabric8io/fabric8-ux/milestone">Milestones</a>
+                    </li>
+                    <li>
+                        <a href="invision-links.html">Designs</a>
```

Comment:
I've updated the navigation to have the Design tab first. The site will still load the repo information and installation instructions first.